snippets/firestore/firestore_pipelines.py

```diff
@@ -0,0 +1,1289 @@
+    for res in result:
+        print(res)
+
+def multiply_function():
+    # [START multiply_function]
+    result = client.pipeline() \
+        .collection("books") \
+        .select(Field.of("price").multiply(Field.of("soldBooks")).as_("revenue")) \
+        .execute()
+    # [END multiply_function]
+    for res in result:
+        print(res)
+
+def divide_function():
+    # [START divide_function]
+    result = client.pipeline() \
+        .collection("books") \
+        .select(Field.of("ratings").divide(Field.of("soldBooks")).as_("reviewRate")) \
+        .execute()
+    # [END divide_function]
+    for res in result:
+        print(res)
+
+def mod_function():
+    # [START mod_function]
+    display_capacity = 1000
+    result = client.pipeline() \
+        .collection("books") \
+        .select(Field.of("unsoldBooks").mod(display_capacity).as_("warehousedBooks")) \
+        .execute()
+    # [END mod_function]
+    for res in result:
+        print(res)
+
+def ceil_function():
+    # [START ceil_function]
+    books_per_shelf = 100
+    result = client.pipeline() \
+        .collection("books") \
+        .select(
+            Field.of("unsoldBooks").divide(books_per_shelf).ceil().as_("requiredShelves")
+        ) \
+        .execute()
+    # [END ceil_function]
+    for res in result:
+        print(res)
+
+def floor_function():
+    # [START floor_function]
+    result = client.pipeline() \
+        .collection("books") \
+        .add_fields(
+            Field.of("wordCount").divide(Field.of("pages")).floor().as_("wordsPerPage")
+        ) \
+        .execute()
+    # [END floor_function]
+    for res in result:
+        print(res)
+
+def round_function():
+    # [START round_function]
+    result = client.pipeline() \
+        .collection("books") \
+        .select(Field.of("soldBooks").multiply(Field.of("price")).round().as_("partialRevenue")) \
+        .aggregate(Field.of("partialRevenue").sum().as_("totalRevenue")) \
+        .execute()
+    # [END round_function]
+    for res in result:
+        print(res)
+
+def pow_function():
+    # [START pow_function]
+    googleplexLat = 37.4221
+    googleplexLng = -122.0853
+    result = client.pipeline() \
+        .collection("cities") \
+        .add_fields(
+            Field.of("lat").subtract(googleplexLat)
+                .multiply(111)  # km per degree
+                .pow(2)
+                .as_("latitudeDifference"),
+            Field.of("lng").subtract(googleplexLng)
+                .multiply(111)  # km per degree
+                .pow(2)
+                .as_("longitudeDifference")
+        ) \
+        .select(
+            Field.of("latitudeDifference").add(Field.of("longitudeDifference")).sqrt()
+                # Inaccurate for large distances or close to poles
+                .as_("approximateDistanceToGoogle")
+        ) \
+        .execute()
+    # [END pow_function]
+    for res in result:
+        print(res)
+
+def sqrt_function():
```

Comment:
This seems like a duplicate of the above? If that's really what you want, I think you can stack multiple region tags over the same sample

Reply:
My understanding is no, and I've definitely seen other samples that double up like that. (Here's [an example](https://github.com/googleapis/python-logging/blob/a7a28d1b9396e472f153fd76d0d846338c107d5a/samples/snippets/handler.py#L20))

But if you want to keep them separate, that's fine